db/dcrpg/queries.go

```diff
@@ -1042,7 +1044,7 @@ func RetrieveAddressSpentUnspent(db *sql.DB, address string) (numSpent, numUnspe
 		log.Debug("Merged debit spent count is not valid")
 	}
 
-	err = dbtx.Rollback()
+	err = dbtx.Commit()
```

Comment:
This is the only part that is bug fix in this PR.  It is baffling to me that we had Rollback here.  Perhaps because the preceding statements don't modify the tables...?